usr/share/rear/backup/DUPLICITY/default/500_make_duplicity_backup.sh

```diff
@@ -73,26 +68,25 @@ if [ "$BACKUP_PROG" = "duplicity" ] ; then
     # maybe better done in an if or case statement
     #
     if [[ $BKP_URL == ssh://* ]] || [[ $BKP_URL == rsync://* ]] || [[ $BKP_URL == fish://* ]] ; then
-		LogPrint "Checking backup-path at server ..."
-		ssh ${DUPLICITY_USER}@${DUPLICITY_HOST} "test -d ${DUPLICITY_PATH}/${HOSTNAME} || mkdir -p ${DUPLICITY_PATH}/${HOSTNAME}"
-	fi
-	
+        LogPrint "Checking backup-path at server ..."
+        ssh ${DUPLICITY_USER}@${DUPLICITY_HOST} "test -d ${DUPLICITY_PATH}/${HOSTNAME} || mkdir -p ${DUPLICITY_PATH}/${HOSTNAME}"
+    fi
+
     # first remove everything older than $BACKUP_DUPLICITY_MAX_TIME
     if [ -n "$BACKUP_DUPLICITY_MAX_TIME" ] ; then
-		LogPrint "Removing the old stuff from server with CMD:
-	$DUPLICITY_PROG remove-older-than $BACKUP_DUPLICITY_MAX_TIME -v5 $BKP_URL/$HOSTNAME"
-		$DUPLICITY_PROG remove-older-than $BACKUP_DUPLICITY_MAX_TIME -v5 $BKP_URL/$HOSTNAME >> ${TMP_DIR}/${BACKUP_PROG_ARCHIVE}.log
+        LogPrint "Removing the old stuff from server with CMD:
+    $DUPLICITY_PROG remove-older-than $BACKUP_DUPLICITY_MAX_TIME -v5 $BKP_URL/$HOSTNAME"
+        $DUPLICITY_PROG remove-older-than $BACKUP_DUPLICITY_MAX_TIME -v5 $BKP_URL/$HOSTNAME >> ${TMP_DIR}/${BACKUP_PROG_ARCHIVE}.log
     fi
 
     # do the backup
-    LogPrint "Running CMD: $DUPLICITY_PROG -v5 $DUP_OPTIONS $GPG_KEY $GPG_OPT $EXCLUDES \
-     / $BKP_URL/$HOSTNAME >> ${TMP_DIR}/${BACKUP_PROG_ARCHIVE}.log "
-
     if [[ "$BACKUP_DUPLICITY_GPG_OPTIONS" ]] ; then
+        LogPrint "Running CMD: $DUPLICITY_PROG -v5 $DUP_OPTIONS $GPG_KEY --gpg-options ${BACKUP_DUPLICITY_GPG_OPTIONS} $EXCLUDES / $BKP_URL/$HOSTNAME >> ${TMP_DIR}/${BACKUP_PROG_ARCHIVE}.log "
         $DUPLICITY_PROG -v5 $DUP_OPTIONS $GPG_KEY --gpg-options "${BACKUP_DUPLICITY_GPG_OPTIONS}" $EXCLUDES \
            / $BKP_URL/$HOSTNAME >> ${TMP_DIR}/${BACKUP_PROG_ARCHIVE}.log 2>&1
     else
-        $DUPLICITY_PROG -v5 $DUP_OPTIONS $GPG_KEY $GPG_OPT $EXCLUDES \
+        LogPrint "Running CMD: $DUPLICITY_PROG -v5 $DUP_OPTIONS $GPG_KEY $EXCLUDES / $BKP_URL/$HOSTNAME >> ${TMP_DIR}/${BACKUP_PROG_ARCHIVE}.log "
+        $DUPLICITY_PROG -v5 $DUP_OPTIONS $GPG_KEY $EXCLUDES \
```

Comment:
@Lukey3332 Did you miss the --gpg-options here?

Reply:
The else is only taken if `$BACKUP_DUPLICITY_GPG_OPTIONS` is empty, so its not needed here.